crates/miden-objects/src/transaction/inputs/mod.rs

```diff
@@ -1,74 +1,60 @@
+use alloc::vec::Vec;
 use core::fmt::Debug;
 
 use super::PartialBlockchain;
 use crate::TransactionInputError;
-use crate::account::PartialAccount;
-use crate::block::BlockHeader;
+use crate::account::{AccountCode, PartialAccount};
+use crate::block::{BlockHeader, BlockNumber};
 use crate::note::{Note, NoteInclusionProof};
-use crate::utils::serde::{
-    ByteReader,
-    ByteWriter,
-    Deserializable,
-    DeserializationError,
-    Serializable,
-};
+use crate::transaction::{TransactionArgs, TransactionScript};
 
 mod account;
 pub use account::AccountInputs;
 
 mod notes;
+use miden_processor::AdviceInputs;
 pub use notes::{InputNote, InputNotes, ToInputNoteCommitments};
 
-// TRANSACTION INPUTS
+// TRANSACTION KERNEL INPUTS
 // ================================================================================================
 
 /// Contains the data required to execute a transaction.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct TransactionInputs {
+pub struct TransactionKernelInputs {
```

Comment:
I think we can keep this as just `TransactionInputs` (since we don't have any other transaction inputs AFAICT).